perfkitbenchmarker/benchmarks/mongodb_ycsb_benchmark.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run YCSB against MongoDB.
+
+YCSB is a load generator for many 'cloud' databases. MongoDB is a NoSQL
+database.
+
+MongoDB homepage: http://www.mongodb.org/
+YCSB homepage: https://github.com/brianfrankcooper/YCSB/wiki
+"""
+
+import os
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker.packages import ycsb
+
+# See http://api.mongodb.org/java/2.13/com/mongodb/WriteConcern.html
+flags.DEFINE_string('mongodb_writeconcern', 'safe',
+                    'MongoDB write concern.')
+
+FLAGS = flags.FLAGS
+
+
+BENCHMARK_INFO = {'name': 'mongodb_ycsb',
+                  'description': 'Run YCSB against MongoDB.',
+                  'scratch_disk': True,
+                  'num_machines': 2}
+
+RESULT_REGEX = r'\[(\w+)\], (\w+)\(([\w/]+)\), ([-+]?[0-9]*\.?[0-9]+)'
```

Comment:
Its helpful to give an example of what you expect to parse correctly.  From this I can tell if it is correct or not.